locksmith/src/operations/userOperations.ts

```diff
@@ -0,0 +1,47 @@
+const ethJsUtil = require('ethereumjs-util')
+const models = require('../models')
+const { User, UserReference } = models
+import * as types from '../types'
+
+namespace UserOperations {
+  export async function createUser(
+    input: types.UserCreationInput
+  ): Promise<Boolean> {
+    let userReference = await UserReference.create(
+      {
+        emailAddress: input.emailAddress.toLowerCase(),
+        User: {
+          publicKey: ethJsUtil.toChecksumAddress(input.publicKey),
+          recoveryPhrase: input.recoveryPhrase,
+          passwordEncryptedPrivateKey: input.passwordEncryptedPrivateKey,
+        },
+      },
+      {
+        include: User,
+      }
+    )
+
+    if (userReference) {
+      return true
```

Comment:
I think we can just `return userReference`?

Reply:
At the moment method's return type is `Promise<Boolean>` we could change it to be an any type but I am not sure that it would provide a lot of value to downs stream consumers in this fashion

Reply:
good point